modules/determining-upgrade-viability-conditiontype.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * updating/index.adoc
+
+:_content-type: CONCEPT
+[id="Understanding_clusteroperator_conditiontypes_{context}"]
+= Understanding cluster operator condition types
+
+The status of cluster operators is revealed with condition type, informing you of potential impacts to upgrading your cluster. 
```

Comment:
Suggested rewrite: The status of cluster operators is shown through conditions, which inform you of potential impacts to upgrading your cluster.

Reply:
We're in the updates section at the moment, so I don't mind "potential impacts to upgrading your cluster".  But conditions are more generically about component status, and not only scoped to updates (e.g. `Upgradeable` is scoped to thoughts about future updates, but `Available` is interesting regardless of whether a component is updating or considering future updates).  [The API docs][1] have:

> ... represents the state of the operator's managed and monitored components.

[1]: https://github.com/openshift/api/blob/cce310ad2932f6de24491052d506926e484c082c/config/v1/types_cluster_operator.go#L112-L113